src/cli/commands/install.js

```diff
@@ -431,6 +431,25 @@ export class Install {
       this.scripts.setArtifacts(artifacts);
     }
 
+    if (Object.getPrototypeOf(this) === Install.prototype) {
+      const registryNames = Object.keys(registries);
+      for (let i = 0; i < registryNames.length; i++) {
+        const {filename} = registries[registryNames[i]];
+        const loc = path.join(this.config.cwd, filename);
+
+        // `fs.exists()` is deprecated since nodejs v8
+        try {
+          await fs.access(loc);
```

Comment:
looks a bit radical, I think this may be done coming from another angle.
If package.json does not exist then patterns from fetchRequestFromCwd call would be empty.